### PR TITLE
Add cstring and math includes, use pow

### PR DESCRIPTION
**Pull request by desttinghim**
I had to make these changes to get the LabSoundGraphToy project to compile on my NixOS machine. I haven't put too much thought into how this *should* be fixed, but I figured I'd share my changes.

### CODE REVIEW

File: src/LabMidiSong.cpp [hunk 1]
```diff
@@ -445,7 +446,7 @@ MidiEvent* parseEvent(uint8_t const*& dataStart, uint8_t lastEventTypeByte)
                 auto event = new Event_TimeSignature();
                 double num = double(*dataStart++);
                 double denom = double(*dataStart++);
-                event->timeSignature = num / std::powf(2., denom);
+                event->timeSignature = num / std::pow(2., denom);
```
Comment (meshula) [inline src/LabMidiSong.cpp:448]: this is probably the fix here, right? cmath defines std::pow, but math.h defines plain ol' pow, so the math.h include is unnecessary?

**Comment by meshula** [comment]
Thanks for the catch ~ is the math.h include necessary?

**Comment by desttinghim** [comment]
It looks like `math.h` include is not necessary. It seems `powf` wasn't in libc on unixes until C11, and before that it was a Microsoft-ism. `powf` and `pow` are slightly different - `powf` uses floats, while `pow` uses doubles.

**Comment by meshula** [comment]
Thanks!

**Pull request merged**